NEWS.md

```diff
@@ -1,5 +1,76 @@
-## 12.7.0 - Unreleased
-* [MODORDERS-908](https://issues.folio.org/browse/MODORDERS-908) - Obtain organisations with pagination
+## 12.8.0 - Unreleased
```

Comment:
@azizbekxm, data-import-processing-core is released so you can use release version now